curation_concerns-models/app/jobs/audit_job.rb

```diff
@@ -24,15 +21,11 @@ def run
     log = run_audit
     fixity_ok = (log.pass == 1)
     unless fixity_ok
-      # send the user a message about the failing audit
-      login = generic_file.depositor
-      user = User.find_by_user_key(login)
-      logger.warn "User '#{login}' not found" unless user
-      job_user = User.audituser()
-      file_title = generic_file.title.first
-      message = "The audit run at #{log.created_at} for #{file_title} (#{uri}) failed."
-      subject = FAIL
-      job_user.send_message(user, message, subject)
+      if CurationConcerns.config.respond_to?(:send_audituser_failure_message)
```

Comment:
Why `send_audituser_failure_message` instead of `after_audit_failure` or `audit_failure_action`?